salt/hg/config/hg.apache.conf.jinja

```diff
@@ -1,3 +1,9 @@
+<IfModule mod_qos.c>
```

Comment:
I guess we just need to make sure `libapache2-mod-qos` is installed from apt and enabled via a state (as referenced in other comment).

Reply:
Ope! missed that it was! This is resolved.